src/constants.js

```diff
@@ -29,3 +29,13 @@ export const IS_MOBILE = (() => {
 
 export const IS_IOS =
     /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
+
+let tempAnchor = document.createElement('a');
+
+export const IS_AR_QUICKLOOK_CANDIDATE = Boolean(
```

Comment:
Could we wrap the anchor in a closure? e.g.:

```js
expost const IS_AR_QUICKLOOK_CANDIDATE = (() => {
  let anchor = document.createElement('a');
  // ...
  return Boolean(...);
})();
```

Reply:
Aren't modules a closure already? 🤔

Reply:
Yeah I think the advantage of a closure would be to memoize the value and not need to create a `HTMLAnchorElement` however I think this is currently achieved by the the module itself. Why I also set the anchor to be `undefined` to allow the Browser to GC.

I can do this change 💯 but I'm not sure what it adds.

Reply:
Doh I totally read @jsantell's code wrong in the above. Yeah that makes sense.